app/controllers/uploaded_files_controller.rb

```diff
@@ -0,0 +1,69 @@
+class UploadedFilesController < ApplicationController
+  before_action :set_uploaded_file, only: [:show, :edit, :update, :destroy]
+
+  # GET /uploaded_files
+  def index
+    @uploaded_files = UploadedFile.all
+  end
+
+
+  # GET /uploaded_files/1
+  def show
+  end
+
+
+  # GET /uploaded_files/new
+  def new
+    @uploaded_file = UploadedFile.new
+  end
+
+
+  # GET /uploaded_files/1/edit
+  def edit
+  end
+
+
+  # POST /uploaded_files
+  def create
+    @uploaded_file = UploadedFile.new(uploaded_file_params)
+
+    if @uploaded_file.save
+      redirect_to @uploaded_file, notice: 'Uploaded file was successfully created.'
+    else
+      render :new
+    end
+  end
+
+
+  # PATCH/PUT /uploaded_files/1
+  def update
+    if @uploaded_file.update(uploaded_file_params)
+      redirect_to @uploaded_file, notice: 'Uploaded file was successfully updated.'
+    else
+      render :edit
+    end
+
+  end
+
+
+  # DELETE /uploaded_files/1
+  def destroy
+    @uploaded_file.destroy
+    redirect_to uploaded_files_url, notice: 'Uploaded file was successfully destroyed.'
+  end
+
+
+  private
+  # Use callbacks to share common setup or constraints between actions.
+  def set_uploaded_file
+    @uploaded_file = UploadedFile.find(params[:id])
+  end
+
+
+  # Never trust parameters from the scary internet, only allow the white list through.
+  def uploaded_file_params
+    params.require(:uploaded_file).permit(:title, :description, :actual_file, :membership_application)
+  end
+
+
+end
```

Comment:
@tochman -- I'm happy to remove stuff.  I'm finding it hard to know what scaffolded stuff to leave in and what to take out.  Convention = leave it in because most projects need it.  That's why it's become the conventional standard and included in the scaffolds. 

If the standard on this project should be to not include so much of it, then I'd say that the generators & templates should be adjusted accordingly.  That way no one can run into this problem because the scaffolds will create exactly what is needed and no more. 

My sense is that there is some subset of scaffolding that is emphasized in CraftAcademy, and so that is the implicit guideline for this project. But for those of us that weren't in CraftAcademy, in the absence of any other information we can only assume that what is generated in the scaffolded code is acceptable.